### PR TITLE
Add additional resource links from I/O 2016

### PR DESCRIPTION
**Pull request by iainmcgin**
This change is [Reviewable](https://reviewable.io/reviews/openid/appauth-android/82)

### CODE REVIEW

**Comment by iainmcgin** [comment]
+@williamdenniss for review

<details><summary><i>Previously, iainmcgin (Iain McGinniss) wrote…</i></summary>
> #### Add additional resource links from I/O 2016
</details>


---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/openid/appauth-android/82#-:-KIBeQjp_bczxNpyH_1P:-1826320048)*

File: README.md [hunk 1]
```diff
@@ -25,9 +25,13 @@ clients when custom URI scheme redirects are used. The library is friendly to
...
 
-A talk providing an overview of using the library for enterprise single sign-on (produced by
-Google) can be found here: 
-[Enterprise SSO with Chrome Custom Tabs](https://www.youtube.com/watch?v=DdQTXrk6YTk).
+## Codelabs, videos and other resources
+
+- A codelab featuring AppAuth was provided for Google I/O 2016: [Achieving Single Sign-on with AppAuth](https://codelabs.developers.google.com/codelabs/appauth-android-codelab/index.html)
+- A talk providing an overview of using the library for enterprise single sign-on 
+  (produced by Google) can be found here: 
+  [Enterprise SSO with Chrome Custom Tabs](https://www.youtube.com/watch?v=DdQTXrk6YTk).
```
Comment (WilliamDenniss) [inline README.md]: https://youtu.be/DdQTXrk6YTk shorthand

File: README.md [hunk 2]
```diff
@@ -25,9 +25,13 @@ clients when custom URI scheme redirects are used. The library is friendly to
...
+- AppAuth is discussed as part of the ["Your apps at work"](https://www.youtube.com/watch?v=Za0OQo8DRM4) session at Google I/O 2016.
```
Comment (WilliamDenniss) [inline README.md]: Should we skip to 22m56s? https://youtu.be/Za0OQo8DRM4?t=22m56s

**Comment by WilliamDenniss** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/openid/appauth-android/82)*

**Comment by iainmcgin** [comment]
Change now obsolete; this was fixed upstream.

**Pull request closed without merging**